docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -26,6 +26,38 @@ Images such as the background, logo and favicon are now loaded from the 'common'
 import=common/your-theme-name
 ----
 
+= Changes to the Account Console theme customization
+
+If you were previously extending the now deprecated version 2 of the Account Console theme, you will need to update your theme to use the new version 3 of the Account Console theme. The new version of the Account Console theme comes with some changes in regards to how it is customized. To start with a clean slate you can follow the new https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-account-console[customization quickstart].
```

Comment:
Since this is 24 and the quickstarts may change in the future, the 24 branch should be used instead:
https://github.com/keycloak/keycloak-quickstarts/tree/release/24.0/extension/extend-account-console